### PR TITLE
Clipboard [Firefox version < 50]

### LINKED ISSUE #2087: Clipboard
hi, 

In chrome, 

1).  I am able copy table formats data and paste into chat as image

image

But the same not able in Firefox browser 44.

Any settings has to be change in our server?

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core
Closes #2087
I create a hack that fix the problem, I'm not sure if is necessary yet, but ....
I think [this pull request](https://github.com/RocketChat/Rocket.Chat/pull/6272/)  must be done here to.

### CODE REVIEW

File: packages/rocketchat-ui-message/client/messageBox.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 			{{#if allowedToSend}}
 			<div class="message-input border-component-color">
 				<div class="input-message-container content-background-color">
+					<div id='msg_contenteditable' contenteditable='true' style='height: 0; width: 0; overflow: hidden'></div>
```
Comment (graywolf336) [inline packages/rocketchat-ui-message/client/messageBox.html]: Use double quotes here.

**Comment by ggazzo** [comment]
I think that pull request fix this issue #862 to.

**Comment by karlprieb** [comment]
@ggazzo Yes, it wil fix :) We tested this copy-paste yesterday.

**Comment by rodrigok** [comment]
@graywolf336 can you review this again?

**Pull request merged**